providers/base/units/mtd/manifest.pxu

```diff
@@ -0,0 +1,4 @@
+unit: manifest entry
+id: has_mtd
+_name: Does platfrom support Memory Technology Device?
```

Comment:
typo: `platform`. Also, currently this appears like that in the Manifest screen:

```
Does this machine have this piece of hardware?

Does platfrom support Memory Technology Device?     ( ) Yes   ( ) No
```

I think it would be better to just write: "Memory Technology Device (MTD) support" or something like this.

Reply:
Thanks. I modified the name as your suggestion.